test/test_script.py

```diff
@@ -1168,6 +1169,47 @@ def test_cmd_datetime(self):
             areg = r"^Wrong number of arguments for \$datetime: Expected between 0 and 1, "
             with self.assertRaisesRegex(ScriptError, areg):
                 self.parser.eval("$datetime(abc,def)")
+            # The following is disabled because it only raises the exception for Python < 3.7 on Ubuntu
+            # # Tests with invalid format code (ends with % failes on all platforms)
+            # areg = r"^\$datetime:\d+:\d+: Unsupported format code"
+            # with self.assertRaisesRegex(ScriptRuntimeError, areg):
+            #     self.parser.eval(r"$datetime(\%)")
```

Comment:
@rdswift you can now remove this comment I guess

There's a typo in top comment "origninal"

Reply:
Good eye.  I'll have to look into a spell checker for VS Code. :wink:

Reply:
@rdswift I use the [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) extension for vscode, which I think is pretty nice. It does not only check strings and comments but also variable names. So if you have e.g. `metedata = Metadata()` it would highlight that typo.

Reply:
Just installed it.  It is a sweet add-on.  Thanks.